assets/js/modules/tagmanager/setup.js

```diff
@@ -189,51 +233,61 @@ class TagmanagerSetup extends Component {
 				usageContext,
 			};
 
-			let errorCode = false;
-			let errorMsg = '';
 			const { accounts, containers } = await data.get( TYPE_MODULES, 'tagmanager', 'accounts-containers', queryArgs );
```

Comment:
Not for now, but I think we should abandon this datapoint in a future PR and instead only rely on `GET:accounts` and `GET:containers`.

Reply:
That would make it a bit slower to populate but that's probably a good idea for maintainability.

Reply:
Yeah good point, I thought about that too after writing this. Sounds cleaner, but a single route is indeed faster in certain circumstances, especially if we wanted to pre-select a container.